packages/adapters/wagmi/src/client.ts

```diff
@@ -282,49 +282,70 @@ export class WagmiAdapter implements ChainAdapter {
           ReturnType<(typeof UniversalProvider)['init']>
         >
 
-        const siweParams = await this.options?.siweConfig?.getMessageParams?.()
+        const clientId = await provider.client?.core?.crypto?.getClientId()
+        if (clientId) {
+          this.appKit?.setClientId(clientId)
+        }
+
+        let chainId = Number(NetworkUtil.caipNetworkIdToNumber(this.appKit?.getCaipNetwork()?.id))
+        let address: string | undefined = undefined
+        let isSuccessful1CA = false
```

Comment:
Can we do `isSuccessfulOneCa` ?

Reply:
1CA is the naming convention we've been using, I'd rather stick with it.